libs/client/types/src/lib/entities/activity.ts

```diff
@@ -1,9 +1,15 @@
-import { Exercise, User } from '..';
+import { Exercise, Park, User } from '..';
 
 export type Activity = {
   id: string;
   startedAt: Date;
   endedAt: Date;
   exercise: Exercise;
   user: User;
+  park: Park;
 };
+
+export type ActivityChartData = {
+  date: number;
+  coins: number;
+}[];
```

Comment:
Für des würdi a eigene datei in z. B. `@bregenz-bewegt/client/types` maha. weil bei da entities söttan wirklich nur datenbank relevante entities si